res/css/views/rooms/_E2EIcon.pcss

```diff
@@ -37,15 +37,6 @@ limitations under the License.
     }
 }
 
-/* white infill for the transparency */
-.mx_E2EIcon::before {
-    background-color: #ffffff;
-    mask-image: url("$(res)/img/e2e/normal.svg");
-    mask-repeat: no-repeat;
-    mask-position: center;
-    mask-size: 80%;
-}
```

Comment:
Why can this vanish? Is it intentional the icons become hollow?

Reply:
That's my interpretation of the designs linked above, that these icons shouldn't use a hard-coded white fill anymore, as it's contrary to how all of our other icons with symbolic cut-outs appear in dark mode

![Screenshot from 2024-03-14 15-42-41](https://github.com/matrix-org/matrix-react-sdk/assets/48614497/10722bbd-dd5d-421b-b014-2bd1130b8f9e)

Reply:
The design only seems to show the composer tick shield, can we verify with design about the timeline icons and for the other shield variants

Reply:
If the icon came from Compound then it'd be fine but this icon seems to be a legacy one so may want some legacy treatments until it is replaced.

Reply:
Gaëlle has confirmed that this change is intentional